src/content/docs/accounts/accounts-billing/account-setup/create-your-new-relic-account.mdx

```diff
@@ -17,7 +17,7 @@ redirects:
   - /docs/accounts/install-new-relic/account-setup/create-your-new-relic-account
 ---
 
-To create a New Relic account, go to [newrelic.com/signup](https://newrelic.com/signup). Creating a New Relic account gives you free access to our entire [observability platform](https://newrelic.com/platform).
+To create a new New Relic account, go to [newrelic.com/signup](https://newrelic.com/signup). Creating a New Relic account gives you free access to our entire [observability platform](https://newrelic.com/platform).
```

Comment:
"new New"... I know there's nothing really wrong with it, but I just don't like it. 

How about "To create another New Relic account..."?